app/templates/main/reviewproposal.html

```diff
@@ -6,7 +6,7 @@ <h5  class="col-11 modal-title text-center">Review Course Proposal </h5>
           <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
         </div>
         <div class="modal-body">
-          <table class="table table-striped ml-4" id="myTable">
```

Comment:
This id is used by js

Reply:
![among-us-sus](https://user-images.githubusercontent.com/89262886/180026810-7a4f5dc3-a3d3-4066-b6fc-2c50ee5e06a3.gif)